### PR TITLE
Not the end. The start of something bigger!

### PR DESCRIPTION
**Pull request by nastik1811**
Other words, the final milestone.

### CODE REVIEW

**Review by gmltA — APPROVED** [review]
Very good!

File: src/styles/_icons.scss [hunk 1]
```diff
@@ -1,33 +1,50 @@
...
 
-    &:focus{
-        outline: none;
+  }
+  
+  %delete-icon{
+    @extend %icon-btn;
+    background-image: url("../assets/icons/trash.svg");
+    &:hover{
+        background: url("../assets/icons/trash-hover.svg");
```
[the remark this answers is not in the document]
Comment (Nastik1811) [reply src/styles/_icons.scss:48]: Yep

File: src/views/CardForm/CardForm.module.scss [hunk 2]
```diff
@@ -91,20 +79,9 @@ label{
...
-  height: 40px;
-  width: 40px;
-  background-color: transparent;
-  border: none;
-  background-image: url('../../assets/icons/add.png');
-  background-size: contain;
-  transition: 0.05s ease-in;
+@media(prefers-color-scheme: dark){
+  .select{
+    color: white;
```
[the remark this answers is not in the document]
Comment (Nastik1811) [reply src/views/CardForm/CardForm.module.scss:84]: image

[the remark this answers is not in the document]
Comment (Nastik1811) [reply src/views/CardForm/CardForm.module.scss:84]: image

File: src/views/Manager/CollectionsPanel/ConfirmationDialog/index.js [hunk 3]
```diff
@@ -9,16 +9,16 @@ const ConfirmationDialog = ({isOpen, onConfirm, onDismiss, collection}) => {
         <Modal onDismiss={onDismiss}>
                 <ModalHeader title="Confirm deletion" onDismiss={onDismiss}/>
                 <ModalContent>
-                    <ModalMessage> You're gonna delete the <span>{collection.name}</span> collection. </ModalMessage>
+                    <ModalMessage> You're gonna delete the <span>{collection.name}</span> collection. Is that all right?</ModalMessage>
```
[the remark this answers is not in the document]
Comment (Nastik1811) [reply src/views/Manager/CollectionsPanel/ConfirmationDialog/index.js:12]: image
A little mix-up with expressions.

File: src/styles/_icons.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (gmltA) [reply src/styles/_icons.scss:48]: So why don't use `fill` property then?

**Comment by Nastik1811** [comment]
image

**Comment by Nastik1811** [comment]
Hooray
giphy

**Pull request merged**